### DISCUSSION TITLE
Spline gateway showing 404 service not found on Venilla linux setup

### DISCUSSION THREAD

**Discussion by jskalasariya** [AbsaOSS/spline#1294] [Q&A] [answered]
Hello Team,

I am setting up spline services using plain venilla method following official guide - unfortunately docker hub is disabled at org level.
so far I am able to setup ArangoDB and spline db is also created with admin jar. 

Have done tomcat server setup and also changed context.xml with ArangoDB properties. 
Deployed rest-gateway-0.7.8.war to webapps in tomcat.

now on tomcat under base_url/gateway/index.html I can see 404 - service not found.

I have tried looking into logs but nothing helped so far.

Has anyone face such issues and were able to sort this out?
Appreciate your help!!

**Comment by cerveada** [comment, marked as answer]
Are you sure you use the right url? Tomcat will use the war name as url for your application, unless you configure it otherwise.
Try to look into it.

This could also be helpful:
https://www.webucator.com/article/how-to-list-deployed-applications-using-the-apache/

**Reply by jskalasariya** [reply]
yes that was the case I was not using proper url. Thank you
